kanidm_book/src/radius.md

```diff
@@ -165,25 +168,26 @@ You can then run the container with:
 
     docker run --name radiusd -v ...:/data kanidm/radius:latest
 
-Authentication can be tested through the client.localhost NAS configuration with:
+Authentication can be tested through the client.localhost Network Attached Storage (NAS) configuration with:
```

Comment:
It's actually Network Access Server (NAS), because yay namespace collisions!

Reply:
NAS (Lil X).

Reply:
Thanks :) Network Attached Storage didn't make sense, but who knows with these newfangled services.